src/H5F.c

```diff
@@ -2785,3 +2785,119 @@ H5Fset_dset_no_attrs_hint(hid_t file_id, hbool_t minimize)
 done:
     FUNC_LEAVE_API(ret_value)
 } /* H5Fset_dset_no_attrs_hint */
+
```

Comment:
These routines should not be in this file.  They are also specific to the SWMR VFD and should start with H5FDswmr..., same as the MPI-IO, family, etc specific calls for those VFDs.